templates/patient_template.js.erb

```diff
@@ -32,10 +32,13 @@ const QDMPatientSchema = new Schema({
 // to its respective Mongoose Model
 QDMPatientSchema.methods.initializeDataElements = function initializeDataElements() {
   let typeStripped;
+  let elementTypeRemoved;
   const dataElementsInit = [];
   this.dataElements.forEach((element) => {
     typeStripped = element._type.replace(/QDM::/, '');
-    dataElementsInit.push(new AllDataElements[typeStripped](element));
+    elementTypeRemoved = JSON.parse(JSON.stringify(element));
```

Comment:
This is the simplest method of copying this object that I could find. Just wanted to point it out in case someone else has a better suggestion. https://stackoverflow.com/a/10869248

Reply:
This removes functions from `element`, but passing in the json without the functions in line 41 below adds them back.  So this should be OK